src/Features/CSharp/Portable/ConvertAutoPropertyToFullProperty/CSharpConvertAutoPropertyToFullPropertyCodeRefactoringProvider.cs

```diff
@@ -46,24 +48,26 @@ internal override async Task<string> GetFieldNameAsync(Document document, IPrope
         }
 
         internal override (SyntaxNode newGetAccessor, SyntaxNode newSetAccessor) GetNewAccessors(
-            DocumentOptionSet options, SyntaxNode property,
+            CodeGenerationOptions options, SyntaxNode property,
```

Comment:
same poitn abotu generics here.   then you don't need two options objects with different types.